libs/s3/s3_blackbox_test_lib.py

```diff
@@ -32,7 +32,7 @@
 from commons.utils.assert_utils import assert_in
 from commons.constants import S3_ENGINE_RGW
 from config import CMN_CFG
-from config.s3 import S3_CFG, S3_BLKBOX_CFG
+from config.s3 import S3_CFG
 from config.s3 import S3_BLKBOX_CFG as S3FS_CNF
```

Comment:
Can you please explain why this modification is needed?

Reply:
It was imported twice(L35, L36) & Codacy showing medium issue "Reimport 'S3_BLKBOX_CFG' (imported line 35)".
So, removed it from line 35.

Reply:
got it, I missed that line 36 was an existing line!